frontend/packages/shared/src/components/WebLink/WebLink.scss

```diff
@@ -1,6 +1,6 @@
 .weblink {
-    .icon {
-        vertical-align: -0.5em;
-        padding: 0 0.2em;
-    }
+  .icon {
```

Comment:
Pystyykö tätä hyödyntämään luokkana ilman että on weblink-luokan sisältämän komponentin sisällä, ei kai?

Reply:
Joo, pitää olla `weblink` classin alla. Vois tää tietty ollu joku yleinen luokkaki. 🤔

Reply:
Mietin just että ehkä parempi niin että weblink luokan alta tuota hyödynnetään, periaatteessa vois kai käyttää myös tekstin perässä tosiaan ilmankin mut voi hyvin olla ainakin musta näin